src/ansys/units/systems.py

```diff
@@ -193,14 +184,25 @@ def CURRENT(self):
     def CURRENT(self, new_mass):
         self._set_type(unit_type=BaseDimensions.CURRENT, unit=new_mass)
 
-    @property
-    def SOLID_ANGLE(self):
-        """Solid Angle unit of the unit system."""
-        return self._SOLID_ANGLE
+    if os.getenv("PYANSYS_UNITS_ANGLE_AS_DIMENSION"):
+
+        @property
+        def ANGLE(self):
+            """Angle unit of the unit system."""
+            return self._ANGLE
+
+        @ANGLE.setter
+        def ANGLE(self, new_mass):
```

Comment:
`new_mass`. Also below.

Reply:
Need to discuss.

Reply:
I'm saying that there are mass variables here, and I am querying it.

Reply:
We are not handling at this level.